addons/payment_stripe/controllers/main.py

```diff
@@ -51,4 +51,9 @@ def stripe_create_charge(self, **post):
         _logger.info('Stripe: entering form_feedback with post data %s', pprint.pformat(response))
         if response:
             request.env['payment.transaction'].sudo().form_feedback(response, 'stripe')
+        tx_transaction = request.env['payment.transaction'].browse(request.session.get('website_payment_tx_id', False))
```

Comment:
Yeah, this is very weird, indeed. Maybe the trick is to NOT call `/website_payment/transaction` manually and let stripe create the transaction here. I just need it [here](https://github.com/odoo-dev/odoo/blob/saas-11-opw-693421-stripe-nim/addons/website_payment/controllers/main.py#L76), but it is only a matter of adding it in the context, so no biggie.